api/organisations/tasks.py

```diff
@@ -47,6 +42,10 @@
     STARTUP_V2,
     SubscriptionCacheEntity,
 )
+from .task_helpers import (
+    _handle_api_usage_notifications,
```

Comment:
This shouldn't still be a private function since we're calling it from another module now.

Reply:
Ok I've renamed it